glide.lock

```diff
@@ -49,8 +49,4 @@ imports:
   version: ff7ea4431a97bb2b389739cb9087bbc11faa1fe0
   subpackages:
   - cuda/cu
-- name: golang.org/x/sys
```

Comment:
The date needs to be changed at the top of the file if the contents of the .lock change.